data/json/mapgen/occupied_lumber_mill/occupied_lumber_mill.json

```diff
@@ -5,7 +5,7 @@
     "object": {
       "fill_ter": "t_thconc_floor",
       "rows": [
-        "vvvvvvvvvvvvvvvvvv#;;;;;;;;;;#vvvvvvvvvvvvvvvvvv",
+        "vvvvvvvvvvvvvvvvvvV#;;;;;;;;;;#vvvvvvvvvvvvvvvvv",
```

Comment:
This shifts the gate by one tile to the "east", any reason for that? (Just a mistake?)